src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,168 @@
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// Import each theme.
+@import './clinical-lowlight-theme/ExampleComponent.module';
+@import './orion-fusion-theme/ExampleComponent.module';
+
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+The recommended file structure ends up looking like this:
+
+```
+.
+├── example-component
+│   ├── ExampleComponent.jsx
+│   ├── ExampleComponent.module.scss
+│   ├── clinical-lowlight-theme
+│   │   └── ExampleComponent.module.scss
+│   └── orion-fusion-theme
+│       └── ExampleComponent.module.scss
+```
+
+## Webpack
+
+To enable themes, teams must define a `terra-theme.config.js` configuration file. This file is read by the [terra-toolkit postcss loader](https://github.com/cerner/terra-toolkit/tree/master/config/webpack/postcss) to aggregate themes. This file must be defined for theme CSS assets to be included in the webpack bundle. The loader runs automatically for webpack configurations using [terra-toolkit's webpack config](https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js).
+
+```js
+const themeConfig = {
+  theme: 'terra-dark-theme', // The default theme to be enabled on page load.
+  scoped: ['terra-light-theme', 'terra-lowlight-theme'], // An array of scoped themes. Note: Scoped themes do not work in IE 10.
+};
+
+module.exports = themeConfig;
+```
+
+## Example
+
+Toggle the theme from the dropdown at the top of the page to see changes. The full example code can be seen [here](https://github.com/cerner/terra-application/tree/master/src/terra-dev-site/doc/application/how-to.2/example).
+
+<ExampleComponent />
+
+## FAQ
+
+### Are Terra Theme Variables Reusable?
+
+No, theme variables are considered private and should never be used by consuming applications. Do not use Terra theme variables in custom CSS.
+
+### Is It Okay To Override Terra Component Styles?
+
+It is not recommended to override Terra component styles. The exceptions are layout and positioning concerns such as width or margins.
+
+### Do I Have To Use Terra Toolkit's Webpack Configuration?
+
+It is highly recommended that teams use the terra-toolkit webpack configuration to ensure the appropriate loaders are used.
+
+### Can I Use Multiple Themes Providers At Once?
+
+Technically yes, officially no. Most themes depend on the base font-size set on the html element. Only one font-size can be applied to the base html element. Nested theme providers will not apply a correct font-size. This will impact any rem based CSS (padding, margins, line-heights, etc...).
+
+### Can I Create An Entirely New Theme?
+
+No, the internal implementation of Terra components is considered private and changes frequently. Terra provides all supported themes.
```

Comment:
Maybe:

```suggestion
No, the internal implementation of Terra components is considered private and changes frequently. Terra maintains the list of all supported themes.
```

Reply:
I'm pretty sure we provide all supported themes, consumer being the exception.

Reply:
Yeah I'm not really sure how to phrase what I'm trying to say.  I want to make it clear that we don't necessarily provide all the values for custom components.

Reply:
```suggestion
No, because the internal implementation of Terra components change frequently, Theme variables are considered a private implementation details and must be maintained by the Terra team. 
```

Reply:
Tweaked the phrasing: https://github.com/cerner/terra-application/pull/25/commits/bbbd86df8667fc13a0b7c0ee3ca9997e9365209f